src/components/input-time-picker/input-time-picker.tsx

```diff
@@ -43,13 +44,34 @@ import {
   FocusTrapComponent
 } from "../../utils/focusTrapComponent";
 import { FocusTrap } from "focus-trap";
-import { formatTimeString, isValidTime, localizeTimeString } from "../../utils/time";
+import {
+  formatTimeString,
+  isValidTime,
+  localizeTimeString,
+  toISOTimeString
+} from "../../utils/time";
 import { Scale } from "../interfaces";
 import { TimePickerMessages } from "../time-picker/assets/time-picker/t9n";
 import { connectMessages, disconnectMessages, setUpMessages, T9nComponent } from "../../utils/t9n";
 import { InputTimePickerMessages } from "./assets/input-time-picker/t9n";
 import { CSS } from "./resources";
 
+import dayjs from "dayjs";
+import customParseFormat from "dayjs/plugin/customParseFormat";
+import localeData from "dayjs/plugin/localeData";
+import localizedFormat from "dayjs/plugin/localizedFormat";
+import preParsePostFormat from "dayjs/plugin/preParsePostFormat";
+import updateLocale from "dayjs/plugin/updateLocale";
+
+dayjs.extend(customParseFormat);
+dayjs.extend(localeData);
+dayjs.extend(localizedFormat);
+dayjs.extend(preParsePostFormat);
+dayjs.extend(updateLocale);
+
+// This dayjs global is needed by the lazy-loaded locale files
+(window as any).dayjs = dayjs;
```

Comment:
Taking a look to see if we can avoid this. :eyes:

Reply:
yes!  Appreciate any help on figuring out another way :)